tests/conftest.py

```diff
@@ -0,0 +1,156 @@
+import json
+import os
+import random
+import shutil
+import string
+import subprocess
+import time
+import urllib.request
+
+import pytest
+from ephemeral_port_reserve import reserve
+
+from docker_push_latest_if_changed import _inspect_image
+
+
+DOCKER_REGISTRY_IMAGE_NAME = 'registry'
+DEFAULT_TESTING_IMAGES_PATH = os.path.abspath(
+    os.path.join('testing', 'dockerfiles')
```

Comment:
probably should join with `os.path.dirname(__file__)` so that this test doesn't depend on where you invoke it from

Reply:
Sounds good to me.